### DISCUSSION TITLE
How to create scalar array on postgres

### DISCUSSION THREAD

**Discussion by lomegg** [prisma/prisma#9399] [Q&A] [answered]
Hi! Please clarify how exactly I should create integer array field on postgres using prisma. Thanks.

```
Error: Schema parsing
error: Type "Integer" is neither a built-in type, nor refers to another model, custom type, or enum.
  -->  schema.prisma:44
   | 
43 |     might       Int 
44 |     mightNodes  Integer [] 
```

**Comment by ryands17** [comment, marked as answer]
@lomegg 👋 

The same type is used for a scalar array:

```prisma
model ModelName {
  mightNodes Int[]
}
```

**Reply by lomegg** [reply]
Thanks. I tried exactly that before but had another error because of `prismix` module. It wasn't able to determine usage of `postgres `in a separate file.
In a singular schema file it works just fine.

**Reply by ryands17** [reply]
Unfortunately I have no idea about `prismix`. It would be best to ask this in the `prismix` repo.

**Reply by lomegg** [reply]
No worries, I decided agains using it in favor of using simple `fs.createReadStream(file).pipe(outStream)` for combining multple prisma files in a single schema. Thank you!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
